### ISSUE TITLE
Investigate L10N solutions for landing pages

### ISSUE THREAD

**Issue by brettgaylor** [MozillaFoundation/Advocacy#4]
cc @ScottDowne - are we currently using any frameworks for our react-based landing pages?  ie one that would work on the /encrypt page?

**Comment by ScottDowne** [comment]
TLDR Yea, react-intl + pontoon is used heavily on donate, which is likely our most localized project, and only localized project written in react that I know of. This has been battle tested in donate for last year's end of year campaign.

[react-intl](https://github.com/yahoo/react-intl) is just a framework, nothing fancy.

This is [pontoon with donate](https://pontoon.mozilla.org/en-GB/fundraising/One%20page/) and how contributors update the languages, it's super cool and you should check it out if you havn't yet. It automatically creates pull requests in github.

/encrypt and advocacy.mozilla.org is built on the same foundation as donate, so the integration steps are the same. No blockers, we just need to prioritize it. My guess is a week or two of work including translations.

The thing with l10n is the workload for adding react-intl is 90% of the work, actually translating one page or six is a pretty insignificant difference in work. Same goes for adding a few new languages.

How many languages do we want to start with? We can request languages from the community as soon as we're ready, and donate has also used contextor for paid translations in a pinch. It's worth it for donate to pay if it means a day or two sooner of a new locale could be thousands. We'll also get random translations by doing nothing over time.

Anyway, probably more info that you need right now :P but it's pretty cool stuff :)

**Comment by brettgaylor** [comment]
Great, thanks for all that detail.

This will likely be just for one language (Spanish), and for just a few pages (a landing page and a pledge page).

So once we have the design from https://github.com/MozillaFoundation/Advocacy/issues/5 we can start to scope out a new build that includes translation.  Does that seem like the right steps?

**Comment by ScottDowne** [comment]
@brettgaylor works for me.

**Comment by brettgaylor** [comment]
Closing and will file implementation ticket
